automata/base/config.py

```diff
@@ -0,0 +1,9 @@
+"""Global configuration for the library and its primitives"""
+
+
+class Config():
```

Comment:
If the only configuration is the global configuration, then I'd argue there shouldn't be a class, just variables in the module. Instantiation is unnecessary here.

Reply:
@Tagl You're right. I thought I would need some sort of container object so that the module importing the config doesn't just latch onto the initial value of the setting. But that would only be a problem if I import the variable directly. I can use the module itself as said container.

```python
# doesn't work
from automata.base.config import should_validate_automata
if should_validate_automata:
    ...
```

```python
# should work fine
from automata.base import config
if config.should_validate_automata:
    ...
```